bodhi/server/views/generic.py

```diff
@@ -465,6 +465,40 @@ def api_version(request):
     return dict(version=bodhi.server.util.version())
 
 
+@view_config(route_name='api_version', renderer='json')
+def liveness(request):
+    """
+    Return the Bodhi API version.
+
+    Args:
+        request (pyramid.request.Request): The current request.
+    Returns:
+        dict: A dictionary with a "version" key indexing a string of the Bodhi version.
+    """
+    return "ok"
+
+
+@view_config(route_name='api_version', renderer='json')
+def readyness(request):
+    """
+    Return the Bodhi API version.
+
+    Args:
+        request (pyramid.request.Request): The current request.
+    Returns:
+        dict: A dictionary with a "version" key indexing a string of the Bodhi version.
+    """
+    ready = True
+    try:
+        db.session.execute("SELECT 1")
+    except:
+        ready = False
+    if ready:
```

Comment:
I think this if/else statement is unnecessary, all the work can be done inside the try/except statement